spotify-artist-viewer/Models/SAArtistTableViewController.h

```diff
@@ -7,14 +7,10 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "SARequestManager.h"
```

Comment:
Move #import "SARequestManager.h into the .m file because you are not referencing SARequestManager in the .h file.